docs/src/components/version-switcher.tsx

```diff
@@ -4,7 +4,7 @@ import React, { useEffect, useState } from 'react';
 
 export default function VersionSwitcher(): JSX.Element {
   const [versions, setVersions] = useState([]);
-  const [label, setLabel] = useState('Versions');
+  const [activeLabel, setLabel] = useState('Versions');
```

Comment:
```suggestion
  const [activeLabel, setLabel] = useState('v1.127.0');
```
This overrides the fallback so that `v1.127.0` will appear selected. It's a good way to demo what this looks like.